sgx-tools/get-ias-report.go

```diff
@@ -101,7 +101,7 @@ For example, get remote attestation report from IAS according to quote file:
 
 		// get IAS remote attestation report
 		p := parseAttestParameters(spid, subscriptionKey, product)
-		challenger, err := attestation.NewChallenger("sgx-epid", p)
+		challenger, err := registration.NewChallenger("sgx-epid", p)
```

Comment:
NewChallenger is exported by libenclave/attestation/registration rather than libenclave/attestation?

Reply:
> NewChallenger is exported by libenclave/attestation/registration rather than libenclave/attestation?

The pr depends on #608 ,  the codes is out-of-date.

Reply:
> NewChallenger is exported by libenclave/attestation/registration rather than libenclave/attestation?

I'll sync up to #608  when #608 is ready, So just ignore this pr now.